tests/AulaTests.hs

```diff
@@ -163,5 +166,11 @@ boundary mn mx = frequency
     , (98, choose (mn, mx))
     ]
 
+nonEmptyST :: Gen ST
+nonEmptyST = (cs :: String -> ST) . getNonEmpty <$> arb
+
 nonEmptyMarkdown :: Gen Document
-nonEmptyMarkdown = Markdown . (cs :: String -> ST) . getNonEmpty <$> arb
+nonEmptyMarkdown = Markdown <$> nonEmptyST
+
+validTopicDescription :: Gen Description
+validTopicDescription = Description . Text.take Constant.topicDescMaxLength <$> nonEmptyST
```

Comment:
In the `Arbitrary` module many we generate data with noise. I don't want to start clean that module in different PRs. This generator is used in form validation, that is the reason to be.